repositories.json

```diff
@@ -21,6 +21,11 @@
             "name": "first-bit",
             "link": "https://github.com/BitByte-TPC/first-bit",
             "labels": ["first-timers-only", "good first issue", "help wanted"]
+        },
+        {
+            "name": "Aditya vikram gupta",
```

Comment:
`name` field expects name of the repository.  
And I don't think this particular repo is suitable to be added as there's been no activity for quite some time now.

@priyanshnama @garg3133, what do you guys think?

Reply:
Sir on my repo you can see a project from base to advance level that one project I added sir  here is my repo guptaadi123
this is the first time I am taking part in this fest.

Reply:
yeah, but apart from allowing beginners to make contribution to this repository, it also makes sure to have good active projects that they can contribute. So, yeah that's my view.

I will let Priyansh's decide on this one.

Reply:
I am closing this PR.